app/assets/stylesheets/_commons.css.scss

```diff
@@ -174,6 +161,10 @@ body {
     background: rgb(171, 171, 171);
     padding: 17px;
     border-radius: 7px;
+    margin-top: 0.5rem;
+    position: relative;
+    background: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Property `background` already defined on line 161